.circleci/build-osx-cross.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd /tmp
+git clone https://github.com/tpoechtrager/osxcross.git
+cd osxcross
+curl -#L https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.11.sdk.tar.xz -o tarballs/MacOSX10.11.sdk.tar.xz
```

Comment:
The `-#L` confuses me -- do we not want redirects?

Reply:
`-#L` is two options to curl.. progress bar, and follow redirects.  It doesn't actually comment as seen in the syntax highlighting.

Reply:
Will use the long form option instead to clear any confusion in the future.